docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization.md

```diff
@@ -0,0 +1,355 @@
+                                selector: '.product-category',
+                            },
+                        },
+                    },
+                },
+            },
+        ],
+    }]),
+    data: events.data,
+} %}
+
+{# Expected transformed data format in the console:
+  {
+      ...global data/event metadata,
+      relatedProducts: {
+          // Flattened data from the 'details' attribute
+          name: VALUE, // The value taken from the 'name' selector or attribute
+          price: 'static value', // The fixed 'price' value
+          attributes: [VALUE, VALUE, VALUE, ...], // Array of values collected from elements matching '.attribute-selector'
+          metadata: [
+              {
+                  brand: VALUE, // 'brand' data extracted from the '.metadata-row .product-brand' element
+                  category: VALUE, // 'category' data from the '.metadata-row .product-category' element
+              },
+              {
+                  brand: VALUE, // 'brand' data extracted from the '.metadata-row .product-brand' element
+                  category: VALUE, // 'category' data from the '.metadata-row .product-category' element
+              },
+              ...
+          ]
+      }
+  }
+#}
+```
+
+##### Examples of the feature integration into Spryker demo shops
+
+- [B2C Demo Shop](https://github.com/spryker-shop/b2c-demo-shop/pull/595/files)
+- [B2C Marketplace Demo Shop](https://github.com/spryker-shop/b2c-demo-marketplace/pull/474/files)
+- [B2B Demo Shop](https://github.com/spryker-shop/b2b-demo-shop/pull/542/files)
+- [B2B Marketplace Demo Shop](https://github.com/spryker-shop/b2b-demo-marketplace/pull/490/files)
+
+  
+
+### Update website agreement text
+
+You should update the website agreement text and ask for user consent to have their interactions with the website tracked and sent to Algolia.
+Something similar to
+
+> **User Data analytics**
+>
+> To enhance your experience, we use data and analytics to understand how you interact with our site.
+> By accepting, you allow us to capture anonymous events for personalization, analysis, and continuous improvement of your experience on our platform.
+
+### Test it
+
+1. Deploy to testing environment.
+2. Make sure that Algolia is connected and configured in the Backoffice > Apps.
+
+{% info_block warningBox "Make sure" %}
+
+If you previously had ACP Algolia App connected and used, you will need to disconnect and connect it again with the same Algolia credentials in the ACP App Catalog.
+This action will update your Spryker shop config to be able to send events to Algolia.
+
+{% endinfo_block %}
+
+3. Open Yves, act as a guest and logged-in user, do searches, filter results, open product pages after search, add products to cart, do order placement.
+4. Go to [Algolia Dashboard](https://dashboard.algolia.com/) and open Events from Data Sources section - `https://dashboard.algolia.com/apps/$APP_ID$/events/debugger`.
```

Comment:
The link is pointing towards the events>debugger section. Should i go there?

Reply:
the link has a placeholder APP_ID, so it's not a real link that can be used in the documentation, we can just remove it and write about where to find it in the Algolia 
![image](https://github.com/user-attachments/assets/45f07263-d33e-4ece-a807-49207f1a5a90)